src/index.html

```diff
@@ -15,8 +18,97 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO_main_logo"
+        />
```

Comment:
HTML Content: The `alt` attribute for the image should describe the content of the image. Instead of 'MOYO_main_logo', a more descriptive alternative text would be 'MOYO logo'.